recipe/bld.bat

```diff
@@ -67,85 +67,3 @@ del /f %PREFIX%\Library\COPYING
 if errorlevel 1 exit 1
 del /f %PREFIX%\Library\RELEASE.txt
 if errorlevel 1 exit 1
-
-
-:: The CMake Build process adds a -shared at the end of every exe when you don't
-:: build the static libraries.
-:: We copy the shared executables to a name without the -shared suffix to ensure
-:: they are found by programs that expect them in the standard location
-:: We cannot move the files since the generated CMake files from HDF5 still
-:: expect them to exists with the -shared suffix
-:: https://github.com/conda-forge/hdf5-feedstock/pull/188
-echo Copying %LIBRARY_PREFIX%\bin\h5repart-shared.exe %LIBRARY_PREFIX%\bin\h5repart.exe
-copy %LIBRARY_PREFIX%\bin\h5repart-shared.exe %LIBRARY_PREFIX%\bin\h5repart.exe
```

Comment:
did this accidentally increase the size of these binaries since they are now statically linked? Or has HDF5 changed how the linking works for these binaries.

Reply:
The linking has changed, as kindly confirmed by ajelenak [here](https://github.com/conda-forge/hdf5-feedstock/pull/227#issuecomment-2322069903), referencing the upstream changelog [here](https://github.com/HDFGroup/hdf5/blob/hdf5_1.14.4.3/release_docs/RELEASE.txt#L162-L168).